jwt.js

```diff
@@ -240,15 +240,10 @@ function fastifyJwt (fastify, options, next) {
       if (!token) {
         throw new BadRequestError()
       }
-    } else if ((request.headers && request.headers.authorization) && (!onlyCookie)) {
+    } else if ((request.headers && request.headers.authorization && /^Bearer/i.test(request.headers.authorization)) && (!onlyCookie)) {
```

Comment:
There should be space after Bearer:

```js
// Currently
/^Bearer/i.test("BearerWRONG") // true
```

```suggestion
    } else if (request.headers?.authorization && /^Bearer\s/i.test(request.headers.authorization) && !onlyCookie) {
```

Reply:
And maybe we can extract the RegExp Object to a `const` to further increase performance

Reply:
I see that we do that check below

Resolving

Reply:
I did not put the space as BadRequestError is more accurate than NoAuthorizationInHeaderError in a case of `BearerWRONG` . However i will add the space  if you think it is better. just let me know

Reply:
We simply need that space, because currently, this header passes your checks:

`BearerINVALID <token>`

Reply:
Good point, didn't think about it. Commited. 

Thanks!

Reply:
Seems like the error type changed for certain requests, can you please take a look?

Reply:
Yes, will look into it later today